src/components/Summary/PatientSelectContainer.js

```diff
@@ -0,0 +1,16 @@
+import { connect } from 'react-redux';
+import PatientSelect from './PatientSelect';
+import { setSelectedPatient } from '../../actions/Patients/patientActions'
+const mapStateToProps = (state, props) =>
+    ({
+        patients: state.patients
+    });
+
+const mapDispatchToProps = (dispatch, props) =>
+    ({
+        setSelectedPatient(patient) {

```

Comment:
The `patient` variable here looks like a global reference.  Is this meant to be found on props, as in `props.patient` ?